packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -112,7 +110,21 @@ class _EditProductPageState extends State<EditProductPage> {
                 );
                 if (refreshed ?? false) {
```

Comment:
I know you didn't write it, but I'm sure `refreshed == true` is easier to read than `refreshed ?? false`.